crt_portal/cts_forms/tests/test_api.py

```diff
@@ -180,14 +180,14 @@ def test_response_detail_url(self):
         response = self.client.get(self.url)
         self.assertEqual(response.status_code, 200)
         self.assertTrue("false" in str(response.content))
-        self.assertFalse("true" in str(response.content))
+        self.assertTrue("/api/responses" in str(response.content))
```

Comment:
I'm having trouble understanding why this changed - can you help explain?

Reply:
The response content used to not have the value "true" in it but now it sometimes does because of the new is_user_created field (which is often set to true) so I added a different content check based off of the checks performed elsewhere in this file.

Reply:
ah okay, thanks!

in that case, I'm not quite sure how this tests the response detail url - can you assert that something from SAMPLE_REPORT_1 is present in this response?